tests/codex/sales/testreservations.nim

```diff
@@ -68,7 +68,7 @@ asyncchecksuite "Reservations module":
   test "creating availability reserves bytes in repo":
     let orig = repo.available
     let availability = createAvailability()
-    check repo.available == (orig.u256 - availability.size).truncate(uint)
+    check repo.available == (orig.u256 - availability.freeSize).truncate(uint)
 
   test "can get all availabilities":
```

Comment:
Would probably need to include some tests for getting reservations for a particular Availability. And maybe another test showing that reservations included in a different Availability are not returned.

Reply:
Yeah, should have done that 🙈 Thanks for the catch!